modules/gatk4/haplotypecaller/main.nf

```diff
@@ -8,7 +8,7 @@ process GATK4_HAPLOTYPECALLER {
         'quay.io/biocontainers/gatk4:4.2.6.1--hdfd78af_0' }"
 
     input:
-    tuple val(meta), path(input), path(input_index), path(intervals)
+    tuple val(meta), path(input), path(input_index), path(intervals), path(dragstr_model)
```

Comment:
is the model sample dependent or the same across all?

Reply:
It's sample dependent and needs to be created using https://github.com/nf-core/modules/tree/master/modules/gatk4/calibratedragstrmodel